app/DoctrineMigrations/Version153.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version153 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('ALTER TABLE account DROP COLUMN IF EXISTS opening_date');
+        $this->addSql('ALTER TABLE account DROP COLUMN IF EXISTS closing_date');
+        $this->addSql('ALTER TABLE account DROP COLUMN IF EXISTS opening_date_explanation');
+        $this->addSql('ALTER TABLE account DROP COLUMN IF EXISTS closing_balance_explanation');
+        $this->addSql('ALTER TABLE account DROP COLUMN IF EXISTS closing_date_explanation');
```

Comment:
Is there a chance we'll lose some data with this?

Reply:
columns have been removed lots of time ago, as the UX doesn't require nor used this data anymore. Very unlikely we need that, as old reports have been already submitted and it's not required for new reports